dhis-2/dhis-api/src/main/java/org/hisp/dhis/organisationunit/OrganisationUnit.java

```diff
@@ -282,34 +287,55 @@ public void removeAllUsers()
         users.clear();
     }
 
-    public List<OrganisationUnit> getSortedChildren()
+    public List<OrganisationUnit> getSortedChildren( String sortBy )

```

Comment:
Can we instead of a string use `DisplayProperty.java` ? That will make it "typed" and less error prone.